buildconfig/Jenkins/Conda/run-tests

```diff
@@ -101,16 +101,16 @@ fi
 
 # Run unit tests
 cd $WORKSPACE/build
-# mkdir for ctest log
-mkdir -p ctest_log
+# mkdir for test logs
+mkdir -p test_logs
 if [[ $ENABLE_UNIT_TESTS == true ]]; then
-    run_with_xvfb ctest -j$BUILD_THREADS --no-compress-output -T Test -O ctest_log/$OSTYPE.log --schedule-random --output-on-failure --repeat until-pass:3 $WINDOWS_TEST_OPTIONS $WINDOWS_UNITTEST_TIMEOUT_OPTIONS
+    run_with_xvfb ctest -j$BUILD_THREADS --no-compress-output -T Test -O test_logs/ctest_$OSTYPE.log --schedule-random --output-on-failure --repeat until-pass:3 $WINDOWS_TEST_OPTIONS $WINDOWS_UNITTEST_TIMEOUT_OPTIONS
```

Comment:
This is a desirable name change (having simply `$OSTYPE.log` has confused people in the past, and there is an open issue to change it). However, it will likely break the unit test monitor, so we should leave the file name intact for now.

Reply:
I have opened a PR in the `monitor-dashboard` repo to address this https://github.com/mantidproject/mantid-monitor-dashboard/pull/7